pyproject.toml

```diff
@@ -1,3 +1,20 @@
 [build-system]
 requires = ["setuptools>=60"]
 build-backend = "setuptools.build_meta"
+
+[tool.isort]
+profile = "black"
+skip_glob = [
+    "docs/conf.py",
+    "venv*",
+    "local",
+]
+known_first_party = [
+    "toga_android",
+    "toga_cocoa",
+    "toga_dummy",
```

Comment:
I couldn't find a clear explanation of how isort auto-detects this, so I think it's safer to include `toga` for completeness.